### PR TITLE
Added missing permissions + bumped version numbers

### PR DESCRIPTION
**Pull request by pangea-nv**

### CODE REVIEW

File: storage-file-intel/extension.yaml [hunk 1]
```diff
@@ -42,14 +42,14 @@ apis:
 
 roles:
   - role: storage.admin
-    reason: Allows the extension to store resized images in Cloud Storage
+    reason: Allows the extension to store neaturalized files in Cloud Storage
```
Comment (aravindputrevu) [inline storage-file-intel/extension.yaml:45]: Is this neutralized? A typo?

Comment (pangea-nv) [reply storage-file-intel/extension.yaml]: yes thanks you. good catch

**Review by aravindputrevu — APPROVED** [review]
Looks good to me.

File: firestore-audit-log/functions/src/index.ts [hunk 2]
```diff
@@ -234,14 +234,27 @@ const updateResponse = async (
...
-      source: event.subject,
-      status: event.data.input.metadata.threatVerdict
-    });
-  }
-);
+    try {
+      return await logObject({
+        message: `Malicious file '${event.subject}' neaturalize to path '${event.data.output.outputFilePath}'.`,
+        actor: event.source,
+        action: "Neaturalized",
```
Comment (ihmdevcont) [inline firestore-audit-log/functions/src/index.ts:241]: typo -> Neutralized

**Pull request merged**